src/main/resources/openapi/event.yaml

```diff
@@ -16,27 +16,27 @@ paths:
     post:
       tags:
         - Event
-      summary: Event data received from XHIBIT and CPP through a custom web service to add context to the audio recordings it stores, which allows users to request audio by case and hearing data without needing to know the time or physical location of where the audio was recorded.
+      summary: An Endpoint which allows users to request audio by case and hearing data
```

Comment:
i think this is a generic endpoint for an event coming in, its not specific to audio.
/addAudio would be specifically for this